apps/c100/c100_app/deployment.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: apps/v1
```

Comment:
can you use one of our helm charts instead please?

It provides a standardised setup for all our applications.

chart-base was the one created for this work:
https://github.com/hmcts/chart-base

but it's a bit behind by the looks of it.

The simplest to use would be to just use https://github.com/hmcts/chart-nodejs you can ignore the name as it's just about different default limits

Reply:
No inline charts in the repository please

you shouldn't need to create your own chart, please just use ours

Reply:
@timja 
I have gone through toffee java application which points to hmctspublic helm repo in azure acr, but failed to find any helm chart repo for ruby applications. As you mentioned to use the existing charts, so could you please help me to find the charts for ruby?